modules/osint/domain_reputation.py

```diff
@@ -149,6 +170,7 @@ def module_api(self):
 	if LISTS!=0:
 		presence = len(list(set(BLACKLIST))) / LISTS
 		OUTPUT['absence % on blacklist'] = 100 - (presence*100)
```

Comment:
absenc_on_blacklists

Reply:
Should I change that? That was done by kunal. Sending a commit.

Reply:
yes that's missed